src/cpp/include/openvino/genai/generation_config.hpp

```diff
@@ -189,6 +191,8 @@ extern OPENVINO_GENAI_EXPORTS ov::Property<size_t> rng_seed;
 static constexpr ov::Property<float> assistant_confidence_threshold{"assistant_confidence_threshold"};
 static constexpr ov::Property<size_t> num_assistant_tokens{"num_assistant_tokens"};
 
+static constexpr ov::Property<bool> apply_chat_template{"apply_chat_template"};
```

Comment:
let's add python tests here https://github.com/openvinotoolkit/openvino.genai/blob/master/tests/python_tests/test_generation_config.py#L65 to test that `update_generation_config` takes this property into account

Reply:
added